contracts/pool-manager/sim/stableswap.py

```
@@ -0,0 +1,100 @@
+N_COINS = 3
+xp_ = [1000, 1000, 1000]
+
+def get_D(xp, amp):
+    S: uint256 = 0
```

Ruff (0.8.2)

Undefined name `uint256`

(F821)